diff -Naur digimend-kernel-drivers-9/Makefile digimend-kernel-drivers-9_331/Makefile
--- digimend-kernel-drivers-9/Makefile	2018-12-15 15:48:07.000000000 +0000
+++ digimend-kernel-drivers-9_331/Makefile	2020-04-05 21:12:05.392341451 +0100
@@ -19,7 +19,7 @@
 DKMS_SOURCE_DIR = $(DESTDIR)/usr/src/$(DKMS_MODULES_NAME)-$(PACKAGE_VERSION)
 
 modules modules_install clean:
-	$(MAKE) -C $(KDIR) SUBDIRS=$(PWD) $@
+	$(MAKE) -C $(KDIR) M=$(PWD) SUBDIRS=$(PWD) $@
 
 depmod_conf_install:
 	install -D -m 0644 depmod.conf $(DEPMOD_CONF)
